dashboards-reports/server/routes/utils/constants.ts

```diff
@@ -93,6 +93,8 @@ const ipv6Regex = /(([0-9a-fA-F]{1,4}:){7,7}[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:)
 const localhostRegex = /localhost:([1-9][0-9]{0,3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])/g;
 const iframeRegex = /iframe/g;
 
+export const ALLOWED_HOSTS = /^(0|0.0.0.0|127.0.0.1|localhost|(.*\.)?(opensearch.org|aws.a2z.com))$/;
```

Comment:
This should not have an Amazon URL.

Reply:
This is the allowed hosts for all requests and redirections in chromium. The two hosts listed are used by map tiles visualizations

Reply:
Those 2 are not hosts, rather domains. Perhaps narrow the scope to the specific ones we need like `maps.search-services.aws.a2z.com`?

Reply:
I also think this needs to be configurable because some users will run in intranets. Is there a GH issue/feature request for this ask, I can open one.

Reply:
Thansk @dblock for pointing this one out: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/899/files

It looks like we don't need the a2z one. Could you please confirm, @joshuali925 ? If we can also make the `opensearch.org` more specific that'd be even better.

Reply:
I opened https://github.com/opensearch-project/documentation-website/issues/983 to document what's going on with these URLs.

Reply:
Nit: is `ALLOWED_HOSTS` only used for maps? Maybe split it into `ALLOWED_HOSTS` and `ALLOWED_MAPS_HOST` to make it explicit?

Reply:
@davidlago yes, good idea to narrow down to maybe `(.+\.)?maps.opensearch.org`. Will remove the a2z one if someone is able to confirm in db.'s issue which dashboards versions use which endpoint.

@dblock `ALLOWED_HOSTS` is used for all requests and redirections happening inside chromium. Reporting plugin cannot tell if the request is coming from a map visualization, so I'm not sure if there are values to split them. Will add tests after the a2z issue is addressed